renderer/Image_load.cpp

```diff
@@ -552,11 +552,18 @@ void idImage::GenerateImage( const byte *pic, int width, int height,
 	GL_CheckErrors();
 	GL_SetDebugLabel( GL_TEXTURE, texnum, imgName );
 
-		//Routine test( &uploading );
-		auto start = Sys_Milliseconds();
+	//Routine test( &uploading );
+	auto start = Sys_Milliseconds();
+	if ( GLAD_GL_ARB_texture_storage && !generatorFunction ) {
```

Comment:
An additional troubleshooting cvar would be helpful.
So that we could help a poor guy with an old AMD gpu with buggy drivers =)

Reply:
What's the problem with `generatorFunction` here?

Reply:
Old code. Some of our generated dynamic textures (most prominently currentRenderImage as the main FBO render texture) uses an outdated way for generation via R_RGBA8Image, which in turn uses image->GenerateImage to generate a small dummy texture that is then later overridden when the texture is resized to the actual FBO size. However, glTexStorage makes the texture storage immutable, so you can no longer resize the texture and thus the entire FBO setup is borked.

Yes, this code should be updated - render textures should not be following this code path. But I'm not sure if it's as easy as changing the generator from R_RGBA8Image to FB_RenderTexture, so I didn't want to do this here. Guess I should make a follow up ticket...

Reply:
Ok. I did not think someone changes image size after creation.

Reply:
Yeah, I was surprised, too. Took me a moment of debugging time to understand why using glTexStorage broke rendering :D

Reply:
I also added the cvar.